scripts/manage/landing_page_base.md

```diff
@@ -114,6 +114,9 @@ accounts) must be pushed and applied to your deployment to take effect:
 
 ### Add Spinnaker account for GKE
 
+This script grants the Spinnaker deployment's Google Service Account the required
+IAM GKE roles in the project containing the referenced cluster.
```

Comment:
"GKE IAM roles" or "IAM GKE roles"? Would it make since to link to the GKE IAM page, in case users want to learn more?